<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StabTestParamsDialog</class>
 <widget class="QDialog" name="StabTestParamsDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>428</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Параметры методики</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lblProbeSelector">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Проба</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnAddProbe">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Добавить пробу</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../kernel.qrc">
          <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDeleteProbe">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Удалить пробу...</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../kernel.qrc">
          <normaloff>:/images/delete.png</normaloff>:/images/delete.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QFrame" name="frProbeSelector">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="QListView" name="lvProbes">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblCondition">
           <property name="text">
            <string>Условия проведения</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbConditions"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frProbe">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QFrame" name="frProbeKind">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <property name="leftMargin">
             <number>1</number>
            </property>
            <property name="topMargin">
             <number>1</number>
            </property>
            <property name="rightMargin">
             <number>1</number>
            </property>
            <property name="bottomMargin">
             <number>1</number>
            </property>
            <item>
             <widget class="QLabel" name="lblProbeKind">
              <property name="text">
               <string>Назначение пробы</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cbProbeKind"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frStimul">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="leftMargin">
             <number>1</number>
            </property>
            <property name="topMargin">
             <number>1</number>
            </property>
            <property name="rightMargin">
             <number>1</number>
            </property>
            <property name="bottomMargin">
             <number>1</number>
            </property>
            <item>
             <widget class="QLabel" name="lblStimul">
              <property name="text">
               <string>Видеостимуляция</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cbStimul"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frAutoEnd">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <property name="leftMargin">
             <number>1</number>
            </property>
            <property name="topMargin">
             <number>1</number>
            </property>
            <property name="rightMargin">
             <number>1</number>
            </property>
            <property name="bottomMargin">
             <number>1</number>
            </property>
            <item>
             <widget class="QLabel" name="lblAutoEnd">
              <property name="text">
               <string>Автоматически завершать запись</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="cbAutoEnd">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frTime">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="leftMargin">
             <number>1</number>
            </property>
            <property name="topMargin">
             <number>1</number>
            </property>
            <property name="rightMargin">
             <number>1</number>
            </property>
            <property name="bottomMargin">
             <number>1</number>
            </property>
            <item>
             <widget class="QLabel" name="lblTime">
              <property name="text">
               <string>Длительность записи, сек</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTimeEdit" name="edTime">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="displayFormat">
               <string>mm:ss</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frTimeLatent">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="leftMargin">
             <number>1</number>
            </property>
            <property name="topMargin">
             <number>1</number>
            </property>
            <property name="rightMargin">
             <number>1</number>
            </property>
            <property name="bottomMargin">
             <number>1</number>
            </property>
            <item>
             <widget class="QLabel" name="lblTimeLatent">
              <property name="text">
               <string>задержка привыкания, сек</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTimeEdit" name="edTimeLatent">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="displayFormat">
               <string>mm:ss</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frZeroing">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="leftMargin">
             <number>1</number>
            </property>
            <property name="topMargin">
             <number>1</number>
            </property>
            <property name="rightMargin">
             <number>1</number>
            </property>
            <property name="bottomMargin">
             <number>1</number>
            </property>
            <item>
             <widget class="QLabel" name="lblZeroing">
              <property name="text">
               <string>Проводить центровку</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="cbZeroing">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frScale">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <property name="leftMargin">
             <number>1</number>
            </property>
            <property name="topMargin">
             <number>1</number>
            </property>
            <property name="rightMargin">
             <number>1</number>
            </property>
            <property name="bottomMargin">
             <number>1</number>
            </property>
            <item>
             <widget class="QLabel" name="lblScale">
              <property name="text">
               <string>Масштаб</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cbScale">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../kernel.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>StabTestParamsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>418</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>StabTestParamsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>418</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAddProbe</sender>
   <signal>clicked()</signal>
   <receiver>StabTestParamsDialog</receiver>
   <slot>addProbe()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>430</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnDeleteProbe</sender>
   <signal>clicked()</signal>
   <receiver>StabTestParamsDialog</receiver>
   <slot>deleteProbe()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lvProbes</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>StabTestParamsDialog</receiver>
   <slot>selectProbe(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>655</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edTime</sender>
   <signal>timeChanged(QTime)</signal>
   <receiver>StabTestParamsDialog</receiver>
   <slot>changeTime(QTime)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>625</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>655</x>
     <y>37</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edTimeLatent</sender>
   <signal>timeChanged(QTime)</signal>
   <receiver>StabTestParamsDialog</receiver>
   <slot>changeLatentTime(QTime)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>625</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>655</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbStimul</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>StabTestParamsDialog</receiver>
   <slot>changeStimulIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>625</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>655</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbZeroing</sender>
   <signal>clicked(bool)</signal>
   <receiver>StabTestParamsDialog</receiver>
   <slot>changeZeroing(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>625</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel">
     <x>655</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbScale</sender>
   <signal>activated(int)</signal>
   <receiver>StabTestParamsDialog</receiver>
   <slot>changeScale(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>625</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>655</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbAutoEnd</sender>
   <signal>clicked(bool)</signal>
   <receiver>StabTestParamsDialog</receiver>
   <slot>changeAutoEnd(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>625</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>655</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbConditions</sender>
   <signal>activated(int)</signal>
   <receiver>StabTestParamsDialog</receiver>
   <slot>changeCondition(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>660</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbProbeKind</sender>
   <signal>activated(int)</signal>
   <receiver>StabTestParamsDialog</receiver>
   <slot>changeProbeKind(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>530</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>659</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectProbe(int)</slot>
  <slot>addProbe()</slot>
  <slot>deleteProbe()</slot>
  <slot>selectProbe(QModelIndex)</slot>
  <slot>changeTime(QTime)</slot>
  <slot>changeLatentTime(QTime)</slot>
  <slot>changeStimulIndex(int)</slot>
  <slot>changeZeroing(bool)</slot>
  <slot>changeScale(int)</slot>
  <slot>changeAutoEnd(bool)</slot>
  <slot>changeCondition(int)</slot>
  <slot>changeProbeKind(int)</slot>
 </slots>
</ui>
